lab-justin/legacy.js

```diff
@@ -0,0 +1,115 @@
+//'use strict';
```

Comment:
Don't submit pull requests with commented out code. I should have taken off points for this.